### PR TITLE
Logger has been moved and implemented as an internal lib

### PR DESCRIPTION
**Pull request by senderh55**
Related to #445

### CODE REVIEW

**Review by origranot — CHANGES_REQUESTED** [review]
@senderh55  This looks great!

There is a small comment I added (I guess this line was added by mistake)
Also, please run lint and test in your own env to make sure the CI works.

File: apps/backend/src/shortener/shortener.controller.spec.ts [hunk 1]
```diff
@@ -97,3 +97,4 @@ describe('ShortenerController', () => {
     });
   });
 });
+import { reduce } from 'rxjs';
```
[the remark this answers is not in the document]
Comment (senderh55) [reply apps/backend/src/shortener/shortener.controller.spec.ts:100]: I added it by mistake

**Comment by senderh55** [comment]
Thanks for your feedback. I updated the path in .eslintrc in the logger and fixed the mistake in shortener.controller.spec.

**Comment by origranot** [comment]
@senderh55 Great job!

I had to remove the buildable option from the logger.
Thank you for your contribution!

Closes https://github.com/origranot/reduced.to/issues/445

**Pull request merged**